packages/deployments/contracts/hardhat.config.ts

```diff
@@ -74,6 +74,9 @@ const config: HardhatUserConfig = {
     rando: { default: 3 },
   },
   networks: {
+    hardhat: {
+      allowUnlimitedContractSize: true,
```

Comment:
After merging the latest changes from `amarok` the Connext contract exceeded the max size, so tests failed with `Transaction reverted: trying to deploy a contract whose code is too large`. Added this flag so we can continue running the hardhat tests until we fix the Connext contract size

Reply:
Added a TODO comment https://github.com/connext/nxtp/pull/916/commits/8ef54421a4f6da72c0560236a8b587232a67bfd1